styles.css

```diff
@@ -0,0 +1,72 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500&display=swap');
+
+/* Mobile setup */
+@media (min-width: 400px) {

```

Comment:
- [ ] Good job with the media query but your minimum mobile screen size should be `375`px and make sure that mobile version should be responsive until the desktop screen size.

Reply:
I seem to have confused the max with the min. I would've caught that if I had tested it in mobile version on the browser inspect tool. Now I added min-width to 375px and max-width to 600px. On those ranges, the website acquires the styling.